pilot/lib/pilot/build_manager.rb

```diff
@@ -159,12 +159,18 @@ def update_beta_app_meta(options, build)
     end
 
     def self.truncate_changelog(changelog)
-      max_changelog_length = 4000
-      if changelog && changelog.length > max_changelog_length
-        original_length = changelog.length
+      # Apple documentation on "What's New in this Version" field, the one used for changelog
+      # https://help.apple.com/app-store-connect/?lang=en#/devf29afbb74
```

Comment:
The documentation actually does say ` Limited to 4000 characters.`. Do we want to make a note there that the documentation is wrong and it should really be bytes?